client/src/pages/FlightData/tabs/Quick.js

```diff
@@ -44,27 +44,27 @@ const Quick = () => {
 		>
 			<Column style={{ marginBottom: "1rem" }}>
 				<Row>
-					<Box label="Altitude" content={altitude} />
+					<Box label="Altitude" content={(Math.round(altitude * 100) / 100) + " ft"} />
 					<Row>
-						<Box label="Yaw" content={orientation.yaw} />
-						<Box label="Roll" content={orientation.roll} />
-						<Box label="Pitch" content={orientation.pitch} />
+						<Box label="Yaw" content={(orientation.yaw) + "\u00B0"} />
+						<Box label="Roll" content={(orientation.roll) + "\u00B0"} />
+						<Box label="Pitch" content={(orientation.pitch) + "\u00B0"} />
 					</Row>
 				</Row>
 				<Row>
-					<Box label="Ground Speed" content={groundSpeed} />
-					<Box label="Airspeed" content={airspeed} />
+					<Row>
+						<Box label="Latitude" content={(Math.round(latLong.lat * 100000000) / 100000000) + "\u00B0"} />
+						<Box label="Longitude" content={(Math.round(latLong.lon * 100000000) / 100000000) + "\u00B0"} />
```

Comment:
I was thinking of fancy things using degrees North and degrees West, etc. However, I just decided to leave it at just degrees, because I think this measurement is all we need for the competition.

Reply:
Regarding `toFixed` - never heard of it before, but I'll check it out and possibly use it for the roundings instead.

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toFixed

Reply:
It has been implemented in the most recent commit.